org.eclipse.scout.rt.mail/src/main/java/org/eclipse/scout/rt/mail/smtp/SmtpConnectionPool.java

```diff
@@ -93,9 +96,16 @@ public class SmtpConnectionPool {
   protected final Object m_poolLock = new Object();
   protected final Set<SmtpConnectionPoolEntry> m_idleEntries = new HashSet<>();
```

Comment:
Declare by JavaDoc which of the access to member variables must be guarded via poolLock

Reply:
Added further javadoc in 231195a99e67f0e19d31639e53a5fd6b7b791174